tutorials/containers/kubeapps-on-canonical-kubernetes/kubeapps-on-canonical-kubernetes.md

```diff
@@ -0,0 +1,151 @@
+---
+id: kubeapps-on-canonical-kubernetes
+summary: This tutorial will guide you through the installation of Kubeapps on the Canonical Distribution of Kubernetes.
+categories: containers
+status: published
+tags: kubeapps, kubeless, bitnami, kubernetes
+difficulty: 3
+published: 2017-12-07
+author: Sameer Naik <sameer@bitnami.com>
+feedback_url: https://github.com/kubeapps/kubeapps/issues
+
+---
+
+# Supercharge your Kubernetes cluster with Kubeapps
```

Comment:
I'm not found of using "Supercharge" in the title, I think it's ok in the intro, but the title should be more down-to-earth and practical. Suggestions: "How to use Kubeapps on your Kubernetes cluster", "Install Kubeapps on your Kubernetes cluster"...